bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -574,11 +576,24 @@ spec:
                             fieldPath: metadata.namespace
                     image: quay.io/metallb/metallb-operator:main
                     name: manager
+                    ports:
```

Comment:
we need to disable the cert rotation in case of OLM, as we do below with the webhook server. This is because the signing is handled by OLM itself (while it's not if the operator is deployed with manifests)

Reply:
correct, added an option to disable cert rotation for operator in OLM environment.